{
  "Header": {
    "Original": "CVE-2020-35480 (An issue was discovered in MediaWiki before 1.35.1. Missing users (acc ...)",
    "Line": 147276,
    "ID": "CVE-2020-35480",
    "Description": "(An issue was discovered in MediaWiki before 1.35.1. Missing users (acc ...)"
  },
  "Annotations": [
    {
      "Original": "{DSA-4816-1 DLA-2504-1}",
      "Line": 147277,
      "Type": "xref",
      "Bugs": [
        "DSA-4816-1",
        "DLA-2504-1"
      ]
    },
    {
      "Original": "- mediawiki 1:1.35.1-1",
      "Line": 147278,
      "Type": "package",
      "Package": "mediawiki",
      "Kind": "fixed",
      "Version": "1:1.35.1-1"
    },
    {
      "Original": "NOTE: https://phabricator.wikimedia.org/T120883",
      "Line": 147279,
      "Type": "NOTE",
      "Description": "https://phabricator.wikimedia.org/T120883"
    },
    {
      "Original": "NOTE: https://lists.wikimedia.org/pipermail/wikitech-l/2020-December/094126.html",
      "Line": 147280,
      "Type": "NOTE",
      "Description": "https://lists.wikimedia.org/pipermail/wikitech-l/2020-December/094126.html"
    }
  ]
}